pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTImplementsList.java

```diff
@@ -5,7 +5,18 @@
 
 package net.sourceforge.pmd.lang.java.ast;
 
-public class ASTImplementsList extends AbstractJavaNode {
+import java.util.Iterator;
+
+
+/**
+ * Represents the {@code implements} clause of a class declaration.
+ *
+ * <pre>
+ *  ExtendsList ::= "implements" (TypeAnnotation)* ClassOrInterfaceType
```

Comment:
Most of the grammar won't change much, so that we can start tackling it bit by bit. I've been trying to flesh out the node APIs and documentation a lot for a few months, maybe it should become a concerted effort, with "official" guidelines for documentation and API enrichment